plugins/vpc-tunnel/config/netconfig_test.go

```diff
@@ -0,0 +1,126 @@
+// +build !integration,!e2e
+
+// Copyright 2017 Amazon.com, Inc. or its affiliates. All Rights Reserved.
```

Comment:
copyright year should be `2021` or `2017-2021` if you were renaming the file.

Reply:
Changed in next revision.